Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -0,0 +1,107 @@
+  Buildings.Controls.OBC.CDL.Continuous.Switch swi
+    "Switch between a higher value and a lower value"
+    annotation (Placement(transformation(extent={{60,-10},{80,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant yHigSig(final k=yHig)
+    "Higher value for the output"
+    annotation (Placement(transformation(extent={{-40,40},{-20,60}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant yLowSig(final k=-yLow)
+    "Lower value for the output"
+    annotation (Placement(transformation(extent={{-40,-50},{-20,-30}})));
+  Buildings.Controls.OBC.CDL.Continuous.Subtract controlError
+    "Control error (set point - measurement)"
+    annotation (Placement(transformation(extent={{-76,10},{-56,30}})));
+
+initial equation
+  assert(
+    yHig-yLow>1E-6,
+    "The absulte values of yHig should be different from that of yLow. Check parameters.");
+
+equation
+  connect(swi.y, y)
+    annotation (Line(points={{82,0},{88,0},{88,60},{110,60}},
+                                                            color={0,0,127}));
+  connect(greMeaSet.reference, u_s)
+    annotation (Line(points={{-22,6},{-40,6},{-40,0},{-120,0}},
+                                               color={0,0,127}));
+  connect(greMeaSet.u, u_m)
+    annotation (Line(points={{-22,-6},{-64,-6},{-64,-94},
+          {0,-94},{0,-120}}, color={0,0,127}));
+  connect(yHigSig.y, swi.u1)
+    annotation (Line(points={{-18,50},{50,50},{50,8},{58,
+          8}}, color={0,0,127}));
+  connect(yLowSig.y, swi.u3)
+    annotation (Line(points={{-18,-40},{50,-40},{50,-8},
+          {58,-8}}, color={0,0,127}));
+  connect(On, swi.u2)
+    annotation (Line(points={{110,-80},{52,-80},{52,0},{58,0}},
+        color={255,0,255}));
+  connect(controlError.y, yErr) annotation (Line(points={{-54,20},{94,20},{94,0},
+          {110,0}}, color={0,0,127}));
+  connect(greMeaSet.y, swi.u2)
+    annotation (Line(points={{2,0},{58,0}}, color={255,0,255}));
+  connect(controlError.u1, u_m)
+    annotation (Line(points={{-78,26},{-90,26},{-90,
+          -6},{-64,-6},{-64,-94},{0,-94},{0,-120}}, color={0,0,127}));
+  connect(controlError.u2, u_s)
+    annotation (Line(points={{-78,14},{-80,14},{-80,
+          0},{-120,0}}, color={0,0,127}));
+  annotation (defaultComponentName = "controller",
+        Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+        Rectangle(
+          extent={{-100,-100},{100,100}},
+          lineColor={0,0,127},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-148,154},{152,114}},
+          textString="%name",
+          textColor={0,0,255})}),                                Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(info="<html>
+<p>This block grenerates a real control output signal, y, and a boolean control switch signal, y<sub>On</sub>, as described below:</p>
```

Comment:
there is no output y<sub>On</sub>, but `On`. Please be consistent between documentation and the code.

Reply:
Fixed it